### PR TITLE
Parallel merge sort

### PR DESCRIPTION
**Pull request by Ayush7-BIT**
#128 
#### Brief description of what is fixed or changed
Added code for parallel merge sort..

### CODE REVIEW

**Comment by Ayush7-BIT** [comment]
Kindly review it..

File: Parallel Merge Sort [hunk 1]
```diff
@@ -0,0 +1,37 @@
+public void parallelMergeSort(int[] numbers, int numOfThreads) {
```
Comment (HarsheetKakar) [inline Parallel Merge Sort]: Can you implement it in python please?

File: Priority Queue [hunk 2]
```diff
@@ -0,0 +1,38 @@
+class PriorityQueue(object):
```
Comment (HarsheetKakar) [inline Priority Queue]: you are pushing all commits to the same branch which makes having multiple PR redundant.

File: Priority Queue [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  
+if __name__ == '__main__': 
+    myQueue = PriorityQueue() 
+    myQueue.insert(12) 
+    myQueue.insert(1) 
+    myQueue.insert(14) 
+    myQueue.insert(7) 
+    print(myQueue)             
+    while not myQueue.isEmpty(): 
+        print(myQueue.delete())
```
Comment (HarsheetKakar) [inline Priority Queue]: please put new line at the end of each file

Comment (Ayush7-BIT) [reply Priority Queue]: OK sir..

**Pull request closed without merging**